spec/views/hq/orphans/show_spec.rb

```diff
@@ -1,17 +1,21 @@
 require 'rails_helper'
 
-RSpec.describe "hq/orphans/show.html.haml", type: :view do
+RSpec.describe "hq/orphans/show.html.erb", type: :view do
+  let(:orphan) { FactoryGirl.build_stubbed(:orphan) }
+
   describe 'the orphan exists' do
     before :each do
-      @orphan = build_stubbed :orphan
+      assign(:orphan, orphan)
       render
     end
 
     it 'should show the assigned orphan details' do
-      expect(rendered).to match @orphan.name
+      expect(rendered).to match orphan.name
```

Comment:
I can't decide if this one-line test does enough to be useful at all and whether we might as well get rid of it entirely. Also, the contrast between it and the overly detailed form spec once again makes me wonder about the latter...